test/chainsaw/testmetrics/dynatrace-provider/00-install.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: metrics.keptn.sh/v1beta1
+kind: KeptnMetric
+metadata:
+  name: podtatometric
+  namespace: keptn-system
```

Comment:
the namespace here should be removed otherwise chainsaw install the resource in keptn-system, but then loks for it in the default test namespace